examples/winnip_31604_12061_004_120717_L090_CX_07/winnip_31604_12061_004_120717_L090_CX_07.json

```diff
@@ -4,54 +4,40 @@
   "id": "winnip_31604_12061_004_120717_L090_CX_07",
   "properties": {
     "proj:epsg": 4326,
-    "sat:absolute_orbit": 12061,
-    "datetime": "2012-07-17T14:39:45.732157Z"
+    "datetime": "2012-07-17T14:36:47Z"
   },
   "geometry": {
     "type": "Polygon",
     "coordinates": [
       [
         [
-          -97.683,
-          49.486
+          -98.6699,
```

Comment:
Order seems slightly different 🤔

Reply:
does it matter, provided the set of points is the same ?

Reply:
let's continue this conversation in the other PR